deepspeed/runtime/engine.py

```diff
@@ -729,6 +729,9 @@ def bfloat16_enabled(self):
     def fp16_master_weights_and_gradients(self):
         return self._config.fp16_master_weights_and_gradients
 
+    def bfloat16_enabled(self):
```

Comment:
Now redundant because of [this](https://github.com/microsoft/DeepSpeed/blob/a637cc2cd508381d79a522c98fef0d4e747ef28f/deepspeed/runtime/engine.py#L726-L727).

Reply:
removed